.github/workflows/deploy.yml

```diff
@@ -182,12 +190,20 @@ jobs:
       - name: Build React App
         run: npm install && npm run build
 
+      - name: Get App Version
+        run: echo "app_version=$(npm pkg get version | xargs)" >> "$GITHUB_ENV"
+
       - name: Deploy app build to S3 bucket
         run: |
           mv ./out/[[...app]].html ./out/index.html
-          aws s3 sync ./out/ s3://${{ secrets.AWS_S3_BUCKET_PROD }} --delete
-          PKG_VERSION=$(npm pkg get version | xargs)
-          aws s3 sync ./out/ s3://${{ secrets.AWS_S3_BUCKET_PREPROD }}/$PKG_VERSION --delete
+          aws s3 sync ./out/ s3://${{ secrets.AWS_S3_BUCKET_PREPROD }}/${{ env.app_version }} --delete
 
-      - name: Invalidate CDN Cache
-        run: aws cloudfront create-invalidation --distribution-id ${{ secrets.CLOUDFRONT_DISTRIBUTION_ID}} --paths /static/*
+      - name: Update Cloudfront Origin Path
+        run: |
+          aws cloudfront get-distribution-config --id ${{ secrets.CLOUDFRONT_DISTRIBUTION_ID }} > ${{ env.ORIGINAL_CONFIG_FILENAME }}
+          etag=`jq '.ETag' ${{ env.ORIGINAL_CONFIG_FILENAME }} | xargs`
+          jq '.DistributionConfig | (.Origins.Items[] | select(.Id | contains("website")).OriginPath) = "/${{ env.app_version }}"' ${{ env.ORIGINAL_CONFIG_FILENAME }} > ${{ env.UPDATED_CONFIG_FILENAME }}
```

Comment:
This command replaces the `OriginPath` with the new version, but only returns the `DistributionConfig` element and not the full config.
The `ETag` is retrieved separately to be used in the update CLI command.